content/en/docs/concepts/observability-primer.md

```diff
@@ -0,0 +1,127 @@
+questions about that system without knowing its inner workings. Furthermore,
+allows us to easily troubleshoot and handle novel problems (i.e. “unknown
+unknowns”), and helps us answer the question, "Why is this happening?"
+
+In order to be able to ask those questions of a system, the application must be
+properly instrumented. That is, the application code must emit signals such as
+[traces](/docs/concepts/observability-primer/#distributed-traces),
+[metrics](/docs/concepts/observability-primer/#reliability--metrics), and
+[logs](/docs/concepts/observability-primer/#logs). An application is properly
+instrumented when developers don't need to add more instrumentation to
+troubleshoot an issue, because they have all of the information they need.
+
+[**OpenTelemetry**](/docs/concepts/what-is-opentelemetry) is the mechanism by
+which application code is instrumented, to help make a system observable.
+
+## Reliability & Metrics
+
+**Telemetry** refers to data emitted from a system, about its behaviour. The
+data can come in the form of [Traces](#distributed-traces),
+[Metrics](#reliability--metrics), and [Logs](#logs).
+
+**Reliability** answers the question: “Is the service doing what users expect it
+to be doing?” A system could be up 100% of the time, but if, when a user clicks
+“Add to Cart” to add a black pair of pants to their shopping card, and instead,
+the system keeps adding a red pair of pants, then the system would be said to be
+**un**reliable.
+
+**Metrics** are aggregations over a period of time of numeric data about your
+infrastructure or application. Examples include: system error rate, CPU
+utilization, request rate for a given service.
+
+**SLI**, or Service Level Indicator, represents a measurement of a service's
+behaviour. A good SLI measures your service from the perspective of your users.
+An example SLI can be the speed at which a web page loads.
+
+**SLO**, or Service Level Objective, is the means by which reliability is
+communicated to an organization/other teams. This is accomplished by attaching
+one or more SLIs to business value.
+
+## Understanding Distributed Tracing
+
+To understand Distributed Tracing, let's start with some basics.
+
+### Logs
+
+A **Log** is a timestamped message emitted by services or other components.
+Unlike [Traces](#distributed-traces), however, they are not necessarily
+associated with any particular user request or transaction. They are found
+almost everywhere in software, and have been heavily relied on in the past by
+both developers and operators alike to help them understand system behaviour.
+
+Sample Log:
+
+```text
+I, [2021-02-23T13:26:23.505892 #22473]  INFO -- : [6459ffe1-ea53-4044-aaa3-bf902868f730] Started GET "/" for ::1 at 2021-02-23 13:26:23 -0800
+```
+
+Unfortunately, logs aren't extremely useful for tracking code execution, as they
+typically lack contextual information, such as where they were called from.
```

Comment:
```suggestion
typically lack contextual information, such as where they were called from,
```

Reply:
I just noticed that the line below this one was orphaned, so I removed it. Keeping the original punctuation above.